### PR TITLE
Upgraded to OSPRay 1.1.2

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
retest this please

**Comment by favreau** [comment]
retest this please

**Comment by favreau** [comment]
retest this please

**Comment by favreau** [comment]
retest this please

**Comment by favreau** [comment]
retest this please

**Comment by favreau** [comment]
retest this please

**Comment by favreau** [comment]
retest this please

File: CMake/FindOSPRay.cmake [hunk 1]
```diff
@@ -25,32 +25,33 @@
...
-# guess that OSPRay is installed in a peer directory (if in dev) or in a peer to the ParaView source
+# Guess that OSPRay is installed in a peer directory (if in dev)
 find_path(OSPRAY_ROOT ospray
   HINTS ${PROJECT_SOURCE_DIR}/../OSPRay  ${PROJECT_SOURCE_DIR}/../../../OSPRay
   DOC "OSPRay base directory"
   )
 
 if(NOT OSPRAY_ROOT AND NOT OSPRay_FIND_QUIETLY)
-  message("Could not find OSPRay base directory. Please set OSPRAY_ROOT to the root of your local OSPRay git repository.")
+  message("Could not find OSPRay base directory. Please set OSPRAY_ROOT to the root of your local Brayns git repository.")
```
Comment (tribal-tec) [inline CMake/FindOSPRay.cmake]: It's should still 'OSPray git repository' here, no?

File: CMake/FindOSPRay.cmake [hunk 2]
```diff
@@ -25,32 +25,33 @@
...
+# Guess that embree is installed in a peer directory (if in dev)
+find_path(EMBREE_ROOT embree
+  HINTS ${PROJECT_SOURCE_DIR}/../embree  ${PROJECT_SOURCE_DIR}/../../../embree
+  DOC "Embree base directory"
   )
-if(NOT OSPRAY_ROOT AND NOT OSPRay_FIND_QUIETLY)
-  message("Could not find OSPRay build directory. Please set OSPRAY_ROOT to the directory where OSPRay was built.")
+
+if(NOT EMBREE_ROOT AND NOT Embree_FIND_QUIETLY)
+  message("Could not find Embree base directory. Please set EMBREE_ROOT to the root of your local Brayns git repository.")
```
Comment (tribal-tec) [inline CMake/FindOSPRay.cmake]: again, why Brayns?

File: CMake/FindOSPRay.cmake [hunk 3]
```diff
@@ -25,32 +25,33 @@
...
 endif()
 
 set(OSPRAY_INCLUDE_DIRS
   ${OSPRAY_ROOT}/include
   ${OSPRAY_ROOT}/include/ospray
-  ${OSPRAY_ROOT}/include/ospray/embree/common
-  ${OSPRAY_ROOT}/include/ospray/embree
-  ${OSPRAY_ROOT}/include/ospray/embree/include
-  ${OSPRAY_ROOT}/include/ospray/include
+  ${OSPRAY_ROOT}/include/ospray/SDK
```
Comment (tribal-tec) [inline CMake/FindOSPRay.cmake]: why not ${OSPRAY_ROOT}/include here and use ```#include<ospray/SDK/....h>``` in the code?

File: plugins/engines/ospray/OSPRayFrameBuffer.cpp [hunk 4]
```diff
@@ -21,7 +21,7 @@
 #include "OSPRayFrameBuffer.h"
 
 #include <brayns/common/log.h>
-#include <ospray/common/OSPCommon.h>
+#include <SDK/common/OSPCommon.h>
```
Comment (tribal-tec) [inline plugins/engines/ospray/OSPRayFrameBuffer.cpp]: see above comment. Looks weird here to have top-level SDK folder. Imagine /usr/include/SDK?!? Who knows that this is ospray? :)

File: plugins/engines/ospray/OSPRayScene.h [hunk 5]
```diff
@@ -23,8 +23,13 @@
 
 #include <brayns/common/types.h>
 #include <brayns/common/scene/Scene.h>
-#include <ospray/ospray.h>
-#include <ospray/common/OSPCommon.h>
+
+//#include <ospray_cpp/OSPCommon.h>
```
Comment (tribal-tec) [inline plugins/engines/ospray/OSPRayScene.h]: commented code?

File: CMake/FindOSPRay.cmake (hunk 1 above)
Comment (favreau) [inline CMake/FindOSPRay.cmake]: Done

File: CMake/FindOSPRay.cmake (hunk 2 above)
Comment (favreau) [inline CMake/FindOSPRay.cmake]: Done

File: CMake/FindOSPRay.cmake (hunk 3 above)
Comment (favreau) [inline CMake/FindOSPRay.cmake]: Because I include ospray headers that need those paths.

File: plugins/engines/ospray/OSPRayFrameBuffer.cpp (hunk 4 above)
Comment (favreau) [inline plugins/engines/ospray/OSPRayFrameBuffer.cpp]: Dito

File: plugins/engines/ospray/OSPRayScene.h (hunk 5 above)
Comment (favreau) [inline plugins/engines/ospray/OSPRayScene.h]: Oops. Thx.

**Pull request merged**